tutorials/pages/send-shimmer-tokens-with-iota-sdk.md

```diff
@@ -0,0 +1,148 @@
+# Send Shimmer tokens with iota-sdk in NodeJs
+
+## Introduction
+
+This tutorial shows how to get Shimmer testnet tokens and send them to another address using javascript/typescript.
+
+### What you will learn
+
+- Create a Stronghold account
+- Generate a Shimmer address
+- Get some Shimmer testnet tokens
+- Fetch your balance
+- Send tokens to another address
+
+## Setup your environment
+
+[Install NodeJs](https://nodejs.org/) and update to the latest stable version available.
+
+If you want, you can change the [BIP39 mnemonic](https://en.bitcoin.it/wiki/BIP_0039) and choose a different password, but be sure to set the node URLs to a Shimmer testnet node (we use the nodes provided by IF in this tutorial). For example:
+
+```sh
+# Mnemonics (Don't ever use them to manage real funds!)
+MNEMONIC="endorse answer radar about source reunion marriage tag sausage weekend frost daring base attack because joke dream slender leisure group reason prepare broken river"
+MNEMONIC_2="width scatter jaguar sponsor erosion enable cave since ancient first garden royal luggage exchange ritual exotic play wall clinic ride autumn divert spin exchange"
+# The Wallet database folder used to store account data
+WALLET_DB_PATH="./example-walletdb"
+# The Stronghold snapshot file location used to store secrets
+STRONGHOLD_SNAPSHOT_PATH="./example.stronghold"
+# The password to unlock the Stronghold snapshot file (Don't use it to protect real secrets!)
+STRONGHOLD_PASSWORD="24?drowssap"
+# The node URL to issue transactions with
+NODE_URL="https://api.testnet.shimmer.network"
+# The faucet URL to request test coins from
+FAUCET_URL="https://faucet.testnet.shimmer.network/api/enqueue"
+# The explorer URL to look up transactions, blocks, addresses and more
+EXPLORER_URL="https://explorer.shimmer.network/testnet"
+```
+
+## Create an account
+
+Now that we have all details, let's go through the example code.
+
+After including the needed dependencies, we have have the main function that loads the environment variables from the `.env` file we created earlier. It uses the information to set up [Stronghold](/stronghold.rs/welcome) to store our seed safely.
```

Comment:
You don't mention the env file until here

Reply:
Right. Will add before too.